### PR TITLE
Issue #347 Fix: Content Types shown in All Programs and My Programs table should be the front-end labels configured

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts [hunk 1]
```diff
@@ -429,8 +432,38 @@ export class ListNominatedTextbooksComponent implements OnInit, AfterViewInit, O
     }, _.isUndefined);
   }
 
-
   ngOnDestroy() {
     this.stageSubscription.unsubscribe();
   }
+
+  getAllContentTypes () {
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts:439]: move these functions to program service and pass program id to later one when you want to get the content types of specific program

Comment (twsvaishali) [reply src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts:439]: @niteshkesarkar

File: src/app/client/src/app/modules/core/services/programs/programs.service.ts [hunk 2]
```diff
@@ -469,4 +470,44 @@ export class ProgramsService extends DataService implements CanActivate {
     return this.API_URL(req);
   }
 
+  /**
+   * makes api call to get content types
+   */
+  fetchContentTypeFromApi(): Observable<ServerResponse> {
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/core/services/programs/programs.service.ts:476]: @niteshkesarkar why a separate function for 3 lines of code? use only getContentTypes

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/core/services/programs/programs.service.ts]: @twsvaishali - Combined together.

File: src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts (hunk 1 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts]: @twsvaishali  - Used program service for common functions.

**Pull request merged**